github.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/google/go-github/github"
 )
 
-var _ = github.ScopeAdminOrg
+var X = fmt.Sprintf("%s")
```

Comment:
**[govet]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
missing argument for Sprintf("%s"): format reads arg 1, have only 0 args